task/04-arrays-tasks.js

```diff
@@ -536,7 +630,10 @@ function selectMany(arr, childrenSelector) {
  *   [[[ 1, 2, 3]]], [ 0, 0, 1 ]      => 2        (arr[0][0][1])
  */
 function getElementByIndexes(arr, indexes) {
-    throw new Error('Not implemented');
+
+   let strInd=indexes.map(elem => {return `[${elem}]`} ).join("");
+   
+   return eval(`arr${strInd}`);
```

Comment:
Please, never use `eval` in code

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval